test/model/store_test.dart

```diff
@@ -966,69 +1026,69 @@ void main() {
   });
 
   group('UpdateMachine.poll reload failure', () {
-    late LoadingTestGlobalStore globalStore;
+    late UpdateMachineTestGlobalStore globalStore;
 
-    List<Completer<PerAccountStore>> completers() =>
-      globalStore.completers[eg.selfAccount.id]!;
+    Future<void> prepareReload(FakeAsync async, {
+      required void Function(FakeApiConnection) prepareRegisterQueueResponse,
+    }) async {
+      globalStore = UpdateMachineTestGlobalStore(accounts: [eg.selfAccount]);
 
-    Future<void> prepareReload(FakeAsync async) async {
-      globalStore = LoadingTestGlobalStore(accounts: [eg.selfAccount]);
-      final future = globalStore.perAccount(eg.selfAccount.id);
-      final store = eg.store(globalStore: globalStore, account: eg.selfAccount);
-      completers().single.complete(store);
-      await future;
-      completers().clear();
-      final updateMachine = globalStore.updateMachines[eg.selfAccount.id] =
-        UpdateMachine.fromInitialSnapshot(
-          store: store, initialSnapshot: eg.initialSnapshot());
-      updateMachine.debugPauseLoop();
+      final store = await globalStore.perAccount(eg.selfAccount.id);
+      final updateMachine = store.updateMachine!;
       updateMachine.poll();
```

Comment:
Should this `poll` call be here? It looks like it now duplicates one that `UpdateMachine.load` makes.

Reply:
Indeed; thanks.